cmake/README.md

```diff
@@ -123,15 +123,15 @@ It will generate *nmake* *Makefile* in current directory.
 
 To create *Visual Studio* solution file:
 
-     C:\Path\to\protobuf\cmake>mkdir solution & cd solution
-     C:\Path\to\protobuf\cmake\solution>cmake -G "Visual Studio 12 2013 Win64" ^
+     C:\Path\to\protobuf\cmake\build>mkdir solution & cd solution
+     C:\Path\to\protobuf\cmake\build\solution>cmake -G "Visual Studio 12 2013 Win64" ^
      -DCMAKE_INSTALL_PREFIX=../../../../install ^
      ../..
 
-It will generate *Visual Studion* solution file *protobuf.sln* in current directory.
+It will generate *Visual Studio* solution file *protobuf.sln* in current directory.
 
 If the *gmock* directory does not exist, and you do not want to build protobuf unit tests,
-you need to add *cmake* command argument `-Dprotobuf_BUILD_TESTS=OFF` to disable testing.
+you need to add *cmake* command argument `-DBUILD_TESTING=OFF` to disable testing.
```

Comment:
The flag name is changed to "protobuf_BUILD_TESTS" in https://github.com/google/protobuf/pull/783